src/scripts/main.js

```diff
@@ -1 +1,18 @@
 'use strict';
+
+const largeImg = document.querySelector('#largeImg');
+const smallImages = document.querySelectorAll('.list-item__link');
+
+const imagesLinksArray = Array.from(smallImages);
+
+imagesLinksArray.forEach((item) => {
+  item.getAttribute('href');
+});
+
+smallImages.forEach((item, i) => {
+  item.addEventListener('click', function (e) {
+    e.preventDefault();
+
+    largeImg.setAttribute('src', `${imagesLinksArray[i]}`);
```

Comment:
You are trying to set the src of large Img using imagesLinksArray[i], which is the full <a> element, not the href attribute of the anchor. You need to grab the href attribute instead.